src/navigation/AppStackNavigator.js

```diff
@@ -11,6 +12,7 @@ import {
 import { defaultStackNavigatorConfig } from './defaultStackNavigatorConfig';
 
 import { texts } from '../config';
+import { BookmarkCategoryScreen } from '../screens/BookmarkCategoryScreen';
```

Comment:
from /..screens?

Reply:
changed it in 8a84e9e25ce56257a4c54ca67a967bcc32fcc310